sagemaker-featurestore/feature_store_object_detection_ground_truth.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.    df = create_dataframe_from_manifest('data/output.manifest')
</pre><p>Should we provide a default path here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2822/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Actually, the`output.manifest` file is stored in the data folder so it's the default path.